hardware/national_instruments_x_series.py

```diff
@@ -1919,7 +1911,9 @@ def close_odmr(self):
         return retval
 
     def get_odmr_channels(self):
-        ch = [self._scanner_counter_channels[0]]
+        ch = list()
+        if self._scanner_counter_channels:
+            ch = ch.append(self._scanner_counter_channels[0])
```

Comment:
This will raise an error during activation.

This line will leave `ch == None` due to `list.append()` returning `None`.
Leave out the new assignment to `ch` and just append.